python/mlc_chat/nn/kv_cache.py

```diff
@@ -26,19 +31,25 @@ def attention(self, layer_id: int, q: Tensor, k: Tensor, v: Tensor) -> Tensor:
             _expr=rx.BlockBuilder.current().emit(
                 rx.call_dps_packed(
                     "vm.builtin.paged_attention_kv_cache_attention",
-                    [self._expr, rx.PrimValue(layer_id), q._expr, k._expr, v._expr],
+                    [
+                        self._expr,
+                        rx.PrimValue(layer_id),  # type: ignore[arg-type]
+                        q._expr,
+                        k._expr,
+                        v._expr,
+                    ],
                     out_sinfo=q._expr.struct_info,
                 )
             )
         )
         # pylint: enable=protected-access
 
 
-class FlashInferPagedKVCache(PagedKVCache):
+class FlashInferPagedKVCache(PagedKVCache):  # pylint: disable=too-few-public-methods
     """Paged KV cache using FlashInfer (CUDA) kernels."""
 
-    @staticmethod
-    def create(  # pylint: disable=too-many-arguments
+    def __init__(  # pylint: disable=too-many-arguments
```

Comment:
@MasterJH5574 I replaced `FlashInferPagedKVCache.create` with the default constructor - as a leaf object, it unlikely needs a factory method